From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Wed, 10 Jun 2020 12:56:40 -0700
Subject: [PATCH] plasma-discover-update: Fix rpath link

Otherwise, the installed binary is not correctly linked to
libDiscoverCommon.so.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 update/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/update/CMakeLists.txt b/update/CMakeLists.txt
index 5544790..06a7d73 100644
--- a/update/CMakeLists.txt
+++ b/update/CMakeLists.txt
@@ -1,5 +1,6 @@
 add_executable(plasma-discover-update main.cpp DiscoverUpdate.cpp)
 
+set_target_properties(plasma-discover-update PROPERTIES INSTALL_RPATH ${CMAKE_INSTALL_FULL_LIBDIR}/plasma-discover)
 target_link_libraries(plasma-discover-update Discover::Common KF5::CoreAddons KF5::I18n)
 
 install(TARGETS plasma-discover-update ${INSTALL_TARGETS_DEFAULT_ARGS})
